### PR TITLE
remove redundant builds, use twig 2.4 for lower bound test

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by lsmith77** [comment]
@stof wdyt?

**Comment by iakunin** [comment]
@stof, could you give some feedback on this PR?

It's important for me, cause it blocks https://github.com/liip/LiipThemeBundle/pull/198#issuecomment-496947109

File: .travis.yml [hunk 1]
```diff
@@ -36,14 +23,18 @@ matrix:
...
     - php: 7.3
       env: SYMFONY_VERSION=4.1.*;ASSETIC=skip
+    - php: 7.3
+      env: SYMFONY_VERSION=4.2.*;ASSETIC=skip
     - php: 7.3
       env: SYMFONY_VERSION=@dev;ASSETIC=skip
     - php: 7.0
-      env: TWIG_VERSION=2.7.*
-    - php: 7.1
+      env: SYMFONY_VERSION=3.0.*;TWIG_VERSION=2.4.*
```
Comment (stof) [inline .travis.yml]: why forcing to use the unmaintained 3.0 version of Symfony ?

Comment (lsmith77) [reply .travis.yml]: because its the lower bound in the composer.json

**Pull request merged**